pkg/services/screenshot/screenshot.go

```diff
@@ -322,12 +322,12 @@ func NewUploadingScreenshotService(r prometheus.Registerer, service ScreenshotSe
 		service:  service,
 		uploader: uploader,
 		uploadFailures: promauto.With(r).NewCounter(prometheus.CounterOpts{
-			Name:      "upload_failures",
+			Name:      "upload_failures_total",
```

Comment:
I'd prefix the name with `screenshots_` or something similar so that they all refer to the same area.

Reply:
Then you lose the ability to type `grafana_alerting` and see all the metrics that relate to alerting.

Reply:
If it helps the documentation (https://github.com/grafana/grafana/pull/50425) lists all of the metrics related to images in notifications? We could make it possible to change `subsystem` and then we could have our own screenshot metrics just for alerting?